src/CommonCartridge.js

```diff
@@ -746,118 +682,48 @@ export default class CommonCartridge extends Component {
                 padding="small"
                 background="default"
               >
-                <Router>
-                  <React.Fragment>
-                    <Switch>
-                      <Route
-                        exact
-                        path="/"
-                        render={() =>
```

Comment:
We should consider moving the renders to a component and have them in a views folder or something

Reply:
not a blocker tho, just a comment

Reply:
Would definitely like to see CommonCartridge.js refactored -- it's getting big.